src/stream/stream_backend.rs

```diff
@@ -220,6 +220,75 @@ fn create_udp_stream(
     )));
 }
 
+fn create_rtsp_stream(
```

Comment:
these changes are in the stream namespace and not in manager, as the commit message says.